particle/apps/drupal.md

```diff
@@ -1,22 +1,30 @@
 # Drupal
 
+_For information on getting Drupal theming started in Particle, check the [_Drupal 8 Quickstart Guide_](../getting-started/drupal-8.md)._
+
+**App Root:**
+
 `apps/drupal/`
 
-_For information on getting Drupal theming started in Particle, check the_ [_Drupal 8 Quickstart Guide_](../getting-started/drupal-8.md)_._
+**Templates**
+
+`~templates/` - Drupal twig templates. These often will `include`, `embed`, or `extend` the Twig templates found in Pattern Lab like this:
+
+`{% include "@molecules/branding/branding.twig" with { url: path('<front>') } %}`.
 
-\(REVAMP\) - templates/ - Drupal twig templates. These often will `include`, `embed`, or `extend` the Twig templates found in Pattern Lab like this: `{% include "@molecules/branding/branding.twig" with { url: path('<front>') } %}`. We keep the components in Pattern Lab "pure" and ignorant of Drupal's data model and use these templates to map the data between the two. Think of these as the Presenter templates in the [Model View Presenter](https://en.wikipedia.org/wiki/Model–view–presenter) approach. Also, Drupal Twig templates that have nothing to do with Pattern Lab go here.
+We keep the components in Pattern Lab "pure" and ignorant of Drupal's data model and use these templates to map the data between the two. Think of these as the Presenter templates in the [Model View Presenter](https://en.wikipedia.org/wiki/Model–view–presenter) approach. Also, Drupal Twig templates that have nothing to do with Pattern Lab go here.
 
 ## Drupal integration of design system Twig files
 
```

Comment:
Is it worth noting here that Component Library module gives us the ability to extend, embed and include the `@atomic_thing` label here? This might also be a good place to mention that the `@theme_name` is the default Drupal method to pass templates around within the Drupal theme itself.

Reply:
If someone else who knows a little bit more about the mechanical bits of how that works wants to go ahead and add that in? I'm not 100% how that works as of now